codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/smithy/generators/http/HttpBindingGenerator.kt

```diff
@@ -449,7 +460,9 @@ class HttpBindingGenerator(
      */
     // Rename here technically not required, operations and members cannot be renamed.
```

Comment:
Looking at this comment, I'm wondering if we still need this function at all. What's the purpose of renaming if we don't need to?

Reply:
No clue 👀 
This is used on the deserialization side of the shop, so I've avoided touching it in this PR (apart from the automated formatting).